modules/installing-rhv-insecure-mode.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-custom.adoc
+// * installing/installing_rhv/installing-rhv-default.adoc
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
+
+[id="installing-rhv-insecure-mode_{context}"]
+= Installing {product-title} on {rh-virtualization} in insecure mode
+
+By default, the installer creates a valid CA certificate, prompts you for confirmation, and stores the certificate to use during installation. You do not need to create or install one manually.
+
+Although it is not recommended, you can, if necessary, override this functionality and install {product-title} without verifying a certificate by installing {product-title} on {rh-virtualization} in *insecure* mode.
```

Comment:
I would remove "if necessary," from this sentence. It doesn't serve a purpose.

Although it is not recommended, you can override this functionality....

Otherwise, it lgtm.

Reply:
Fixed.